python/sglang/srt/layers/moe/topk.py

```diff
@@ -122,9 +124,20 @@ def grouped_topk(
     )  # [n, e]
     tmp_scores = scores.masked_fill(~score_mask.bool(), 0.0)  # [n, e]
     topk_weights, topk_ids = torch.topk(tmp_scores, k=topk, dim=-1, sorted=False)
+    if share_fusion:
+        topk_ids[:, -1] = torch.randint(low=num_experts,
+                                        high=num_experts + share_fusion,
```

Comment:
nit: gets a bit confused about the numbers (but if it can run just ignore me, probably I just get it wrong).

* num_experts := scores.shape[1]
* topk_ids := topk(scores) ===> topk_ids has value range from 0 to scores.shape[1]
* Here we want to do a randint that can yield value >scores.shape[1]